frontend/src/components/administration/users/users-list.tsx

```diff
@@ -110,8 +110,8 @@ const UsersList: FC<UsersListProps> = ({ users }) => {
 
       <DialogEditUser
         user={selectedUser}
-        handleClose={handleClose}
-        handleSubmit={handleSubmit}
+        onClose={handleClose}
+        onSave={handleSubmit}
```

Comment:
Usually `onClose` passed as the last one

Reply:
OK, I got it. Done.